stackslib/src/chainstate/nakamoto/coordinator/tests.rs

```diff
@@ -297,6 +302,37 @@ pub fn make_token_transfer(
     stx_transfer_signed
 }
 
+/// Make a token-transfer from a private key
```

Comment:
Comment doesn't match code body

Reply:
Fixed in https://github.com/stacks-network/stacks-core/pull/5321/commits/d901503496a4c91671f9700a0e4df972beb1aafd